core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -334,6 +337,23 @@ def _fit_single(self, X, y, model_base, use_child_oof, time_limit=None, **kwargs
             self.models = [model_base]
         self._add_child_times_to_bag(model=model_base)
 
+    def _get_default_fold_fitting_strategy(self):
+        try:
+            ray = try_import_ray()
+        except Exception:
+            ray = None
+        if not ray:
+            return 'sequential_local'
+        # ray not working properly on macos: https://github.com/ray-project/ray/issues/20084
+        # TODO: re-enable macos once this issue is addressed
+        os_fitting_strategy_map = dict(
+            Darwin='sequential_local',
+            Windows='parallel_local',
+            Linux='parallel_local',
+        )
+        current_os = platform.system()
```

Comment:
if Darwin, can we add a log message that only ever triggers once in an execution (to avoid spam) with a notice saying that ray for MacOS must be explicitly enabled by the user?